.gitlab-ci/packet.yml

```diff
@@ -35,13 +35,13 @@ packet_centos7-flannel-addons:
 packet_centos-weave-kubeadm-sep:
   stage: deploy-part2
   <<: *packet
-  when: on_success
+  when: manual
```

Comment:
What are `only: [triggers]` job for?

Reply:
trigger are for the one only executed via a cron

Reply:
@ant31 that's a failfast thing? because there is GitLab schedules as well that we could `only: [schedules]` ?